server/src/main/java/org/elasticsearch/search/aggregations/metrics/TDigestState.java

```diff
@@ -25,7 +26,7 @@
  * through factory method params, providing one optimized for performance (e.g. MergingDigest or HybridDigest) by default, or optionally one
  * that produces highly accurate results regardless of input size but its construction over the sample population takes 2x-10x longer.
  */
-public class TDigestState {
+public class TDigestState implements Releasable {
```

Comment:
Currently this isn't being closed in productive code. But it's using raw arrays and a NoopCircuitBreaker, so there's no need to.
The next PR will make sure that this uses a real CB, and that it's correctly closed everywhere.